src/lwc/geFormFieldLookup/geFormFieldLookup.js

```diff
@@ -8,22 +9,57 @@ export default class GeFormFieldLookup extends LightningElement {
     @api fieldApiName;
     @api objectApiName;
     @api displayValue = '';
+    @api defaultValue;
     @api label;
     @api required;
     @api id; // unique identifier for this field, used mainly for accessibility
 
 
     @track options = [];
-    @track targetObjectInfo;
     @track value;
     @track targetObjectApiName;
+    @track queryFields;
+    @track valid = true;
 
     /**
      * Retrieve information about the object the lookup points to.
      */
     @wire(getObjectInfo, { objectApiName: '$targetObjectApiName' })
     wiredTargetObjectInfo(response) {
-        this.targetObjectInfo = response;
+        if(response.data) {
+            this.targetObjectInfo = response;
+            this.queryFields = this.getQueryFields();
+        }
+    }
+
+    @wire(getRecord, { recordId: '$defaultValue', fields: '$queryFields'})
+    wiredGetRecord({error, data}) {
+        if(data) {
+            if(typeof this.value === 'undefined') {
+                this.value = this.defaultValue;
+                this.displayValue = data.fields.Name.value;
+            }
+        }
+    }
+
+    /**
+     * Check validity, then update the field with a message if field is invalid.
+     */
+    @api
+    reportValidity() {
+        this.valid = this.checkValidity();
+    }
+
+    /**
+     * Check validity without updating the UI
+     * @returns {boolean}
+     */
+    @api
+    checkValidity() {
+        if(this.required) {
+            return typeof this.value !== 'undefined' && this.value !== null && this.value !== '';
```

Comment:
We could a utility method isEmpty() that we could use across the board.

Reply:
The `utilTemplateBuilder` component has an isEmpty() method. Would it be worth the effort to start up a true `util` component to house these moving forward (isEmpty, isObject, format, etc)?

Reply:
@psadie-sfdo - Short update here, am currently in progress pulling anything that isn't specific to the template builder into a new component named `utilCommon`, and updating references across the project to import those functions from `utilCommon`. Will want your review on it again once that's pushed up here.

Reply:
@bdvorachek - awesome, yeah please keep me posted. Thanks for all the effort!!

Reply:
I ended up omitting a few functions that could be common, but didn't align with what I think our `utilCommon` should be. `dispatch`, `handleError`, `showToast` rely on `ShowToastEvent` from `lightning/platformShowToastEvent`. I think `utilCommon` should be self contained and not import from anywhere. Toasts and other messages could be part of a messaging service component if we want to migrate them somewhere. Open to feedback here.

Also, I think `generateId` warrants a second look. I didn't include it for now.